pkg/reader/notify/notify.go

```diff
@@ -0,0 +1,7 @@
+package notify
+
+import "github.com/cilium/tetragon/api/v1/tetragon"
+
+type NotifyInterface interface {
```

Comment:
This seems to be defined elsewhere also (pkg/observer/listener.go)

Reply:
ah yeah let me move it all over to notify.go it simplifies the cycle import problems.